lib/internal/assert/assertion_error.js

```diff
@@ -42,7 +43,10 @@ const kReadableOperator = {
 const kMaxShortStringLength = 12;
 const kMaxLongStringLength = 512;
 
-const kMethodsWithCustomMessageDiff = ['deepStrictEqual', 'strictEqual', 'partialDeepStrictEqual'];
+const kMethodsWithCustomMessageDiff = new SafeSet()
+  .add('deepStrictEqual')
+  .add('strictEqual')
+  .add('partialDeepStrictEqual');
```

Comment:
```suggestion
const kMethodsWithCustomMessageDiff = new SafeSet(['deepStrictEqual', 'strictEqual', 'partialDeepStrictEqual']);
```
this would be fine fwiw, because SafeSet uses SafeArrayIterator